client/my-sites/plans/current-plan/jetpack-checklist/index.js

```diff
@@ -91,9 +112,12 @@ class JetpackChecklist extends PureComponent {
 				<JetpackChecklistHeader isPaidPlan={ isPaidPlan } />
 
 				<Checklist
+					className="jetpack-checklist"
 					isPlaceholder={ ! taskStatuses }
 					progressText={ translate( 'Your Jetpack setup progress' ) }
 				>
+					{ isEnabled( 'jetpack/checklist/performance' ) &&
+						this.renderSectionTitle( translate( 'Security Tools' ) ) }
```

Comment:
Camel casing titles always feels odd to me and I'd rather see them lowercase but I don't feel strongly about it. Suppose it came from designs?

Reply:
Yes, the _Title Case_ is from the designs at paObgF-89-p2.

Reply:
Haha well you and Jeff are the natives here! :-D 👍